src/main/scala-2.11/markets/orders/orderings/PriceTimeOrdering.scala

```diff
@@ -25,7 +25,7 @@ trait PriceTimeOrdering[T <: Order] extends PriceOrdering[T] with TimeOrdering[T
       -1
     } else if ((order1.price == order2.price) && hasTimePriority(order1, order2)) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Consider using case matching instead of else if blocks](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1508237012/issues/source?bid=2921361&fileBranchId=3005746#l26)